Framework/Beamline/test/DetectorInfoTest.h

```diff
@@ -328,426 +305,15 @@ class DetectorInfoTest : public CxxTest::TestSuite {
         "DetectorInfo: cannot set scan interval with start >= end");
   }
 
-  void test_setScanInterval_sync_async_fail() {
-    DetectorInfo info(PosVec(1), RotVec(1));
-    info.setScanInterval({1, 2});
-    TS_ASSERT_THROWS_EQUALS(info.setScanInterval(0, {1, 2}),
-                            const std::runtime_error &e, std::string(e.what()),
-                            "DetectorInfo has been initialized with a "
-                            "synchonous scan, cannot set scan interval for "
-                            "individual detector.");
-  }
-
-  void test_setScanInterval_async_sync_fail() {
-    DetectorInfo info(PosVec(1), RotVec(1));
-    info.setScanInterval(0, {1, 2});
-    TS_ASSERT_THROWS_EQUALS(info.setScanInterval({1, 2}),
-                            const std::runtime_error &e, std::string(e.what()),
-                            "DetectorInfo has been initialized with a "
-                            "asynchonous scan, cannot set synchronous scan "
-                            "interval.");
-  }
-
-  void test_merge_fail_size() {
-    DetectorInfo a(PosVec(1), RotVec(1));
-    DetectorInfo b(PosVec(2), RotVec(2));
-    a.setScanInterval(0, {0, 1});
-    b.setScanInterval(0, {0, 1});
-    b.setScanInterval(1, {0, 1});
-    TS_ASSERT_THROWS_EQUALS(a.merge(b), const std::runtime_error &e,
-                            std::string(e.what()),
-                            "Cannot merge DetectorInfo: size mismatch");
-  }
-
-  void test_merge_fail_no_intervals() {
```

Comment:
This test was removed because now there it at least always one interval defined by default